test_progress.py is failing in the nightlies

**Describe the bug**
I tried to run the integration tests and I expected them to pass instead test_progress.py fails.

**GEOPM version**
84ba1de0d

**Expected behavior**
```
$ PYTHONPATH=${GEOPM_SOURCE}:${PYTHONPATH} python3 ~/geopm/integration/test/test_progress.py -fv
...
$ echo $?
0
```

**Actual behavior**
```
$ PYTHONPATH=${GEOPM_SOURCE}:${PYTHONPATH} python3 ~/geopm/integration/test/test_progress.py -fv

======================================================================
ERROR: test_linear_progress_triad (integration.test.test_progress.TestIntegration_progress)
Test that the triad region with the geopm_tprof_post() call reports
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/test/geopm/integration/test/test_progress.py", line 210, in test_linear_progress_triad
    triad_post_df = grouped_df.get_group(triad_post_hash)
  File "/home/test/.local/lib/python3.6/site-packages/pandas/core/groupby/groupby.py", line 810, in get_group
    raise KeyError(name)
KeyError: '0xa48bc97'

======================================================================
FAIL: test_zero_progress (integration.test.test_progress.TestIntegration_progress)
Test that all regions that do not contain the geopm_tprof_post()
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/test/geopm/integration/test/test_progress.py", line 262, in test_zero_progress
    self.assertEqual(1, len(uu), msg=err_msg)
AssertionError: 1 != 113 : Expected all zero, group 0x0a48bc97 had [2.02917e-04 2.41633e-03 4.68321e-03 6.92128e-03 9.18123e-03 1.13839e-02
 1.36685e-02 1.59095e-02 1.81691e-02 2.03285e-02 2.25510e-02 2.47834e-02
 2.68641e-02 2.90524e-02 3.12794e-02 3.34897e-02 3.57158e-02 3.78761e-02
 4.01129e-02 4.23278e-02 4.45419e-02 4.66961e-02 4.89403e-02 5.11539e-02
 5.33982e-02 5.55423e-02 5.77914e-02 6.00154e-02 6.22212e-02 6.43882e-02
 6.66158e-02 6.88423e-02 7.10480e-02 7.30851e-02 7.53294e-02 7.75574e-02
 7.97554e-02 8.19291e-02 8.41738e-02 8.63812e-02 8.85987e-02 9.07734e-02
 9.30083e-02 9.52112e-02 9.74573e-02 9.96225e-02 1.01859e-01 1.04084e-01
 1.06312e-01 1.08521e-01 1.10775e-01 1.12990e-01 1.15204e-01 1.17367e-01
 1.19592e-01 1.21674e-01 1.23884e-01 1.26042e-01 1.28288e-01 1.30523e-01
 1.32720e-01 1.34902e-01 1.37146e-01 1.39380e-01 1.41602e-01 1.43775e-01
 1.46065e-01 1.48264e-01 1.50469e-01 1.52633e-01 1.54867e-01 1.57074e-01
 1.59281e-01 1.61483e-01 1.63761e-01 1.66023e-01 1.68277e-01 1.70384e-01
 1.72651e-01 1.74910e-01 1.77172e-01 1.79410e-01 1.81687e-01 1.83938e-01
 1.86217e-01 1.88436e-01 1.90737e-01 1.92983e-01 1.95260e-01 1.97488e-01
 1.99764e-01 2.02017e-01 2.04299e-01 2.06527e-01 2.08808e-01 2.11053e-01
 2.13324e-01 2.15565e-01 2.17704e-01 2.19961e-01 2.22231e-01 2.24468e-01
 2.26756e-01 2.29021e-01 2.31290e-01 2.33529e-01 2.35817e-01 2.38073e-01
 2.40339e-01 2.42562e-01 2.44832e-01 2.47214e-01 2.49760e-01]

--- a/integration/test/test_progress.py
+++ b/integration/test/test_progress.py
@@ -110,7 +110,7 @@ class TestIntegration_progress(unittest.TestCase):
         full_name = [rn for rn in region_names if function_name in rn]
         if len(full_name) != 1:
             raise RuntimeError("Single match not found for {}".format(function_name))
-        return hex(self._report.raw_region(host, full_name[0])['hash'])
+        return f'{self._report.raw_region(host, full_name[0])["hash"]:#010x}'
 
     def check_progress(self, time, progress, expected_max, msg):
         """Check a time series of progress values from a region.

